api/src/main/java/org/apache/iceberg/expressions/ExpressionUtil.java

```diff
@@ -564,6 +575,7 @@ private static String sanitizeDate(int days, int today) {
     return "(date)";
   }
 
+  // TODO(epg): `now` is millisecond resolution; shouldn't this be too?
```

Comment:
You can remove this comment because `now` is not in milliseconds. It is in micros. Otherwise this wouldn't work. See lines 256 and 349.

Reply:
Done.